src/OrbitGl/App.cpp

```diff
@@ -1305,8 +1305,7 @@ bool OrbitApp::IsDevMode() const { return absl::GetFlag(FLAGS_devmode); }
 void OrbitApp::SendDisassemblyToUi(std::string disassembly, DisassemblyReport report) {
```

Comment:
Should we also rename this method?

Reply:
Can you explain why? How would you name that method?

Reply:
Because only seeing the naming, before we were moving the disassembly and now we are showing it? I don't understand the relationship between SendDisassemblyToUi and ShowDisassembly

Reply:
There are multiple functions called `Send*ToUi`, `SendTooltipToUi` for examples. They all have in common that you can call them from any thread - they are thread safe.

Changing the name of `SendDisassemblyToUi` to something else would mean it's not clear anymore that this function belongs to this group of functions which are thread-safe.

I renamed `Disassembly` to `ShowDisassembly` to make the name similar to `ShowSourceCode`.

Reply:
Thanks for explaining it! LGTM